crate.ts

```diff
@@ -52,11 +46,11 @@ export class Crate {
 
   /** Prompts the user how they would like to patch and increments the version accordingly. */
   async promptAndTryIncrement() {
-    console.log(`${this.name} is on ${this.version}`);
+    $.log(`${this.name} is on ${this.version}`);
     const versionIncrement = getVersionIncrement();
     if (versionIncrement != null) {
       await this.increment(versionIncrement);
-      console.log(`Set version to ${this.version}`);
+      $.log(`  Set version to ${this.version}`);
```

Comment:
`$.log` is preferrable because it allows being indented if necessary by something upstream.